12LessonTwelve/homework/css/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin:0;
+}
+
+h1 {
+  text-align: center;
+  font-family: Leckerli One;
```

Comment:
remember to add a backup family after a comma here (like Leckerli One, sans-serif;)